static-build/qemu/Dockerfile

```diff
@@ -0,0 +1,45 @@
+from ubuntu:xenial
+
+ARG QEMU_REPO
+# commit/tag/branch
+ARG QEMU_VERSION
+
+WORKDIR /root/qemu
+RUN apt-get update
+RUN apt-get install -y \
+	    python \
```

Comment:
Nit: You could sort this list of packages for easier inspection/maintenance.

Reply:
done